### ISSUE TITLE
Deprecate this repository

### ISSUE THREAD

**Issue by bassosimone** [bassosimone/libight-build-ios#9]
Already doable since the code to build on iOS is already on the core repo.

**Comment by AntonioLangiu** [comment]
I agree. It would be nice to make a repository like [this](https://github.com/AntonioLangiu/measurement-kit-build-ios) to set up a specifically travis test and see if measurement-kit compile also for iOS.

**Comment by bassosimone** [comment]
I see what you mean! Don't you think it would be possible to modify the existing `travis.yml` of measurement-kit to perform the same checks? As a bonus, one gets all the iOS related checks every time he/she pushes a branch.
